more_itertools/recipes.py

```diff
@@ -355,6 +355,27 @@ def unique_everseen(iterable, key=None):
     Sequences with a mix of hashable and unhashable items can be used.
     The function will be slower (i.e., `O(n^2)`) for unhashable items.
 
+    Remember that ``list`` objects are unhashable - you may want to convert
+    an iterable of ``list`` objects to an iterable of ``tuple`` objects
+    to avoid slowdown. S
+
+        >>> slow_iterable = [1, 2], [2, 3], [1, 2]
```

Comment:
It always confuses me if a tuple is created without the parenthesis. I generally only use this when `return`ing.

Not sure if this is something that has to be changed but I thought it could be helpful to mention it in case others think the same way.